lib/govuk_publishing_components/presenters/contextual_navigation.rb

```diff
@@ -166,6 +166,21 @@ def content_tagged_to_other_step_by_steps?
         step_nav_helper.show_also_part_of_step_nav?
       end
 
+      def content_tagged_to_topical_event?(topical_event_id)
```

Comment:
This is a really minor change feel free to ignore, but I just wondered if it's a bit easier to follow the code when the method is more explicitly named `content_tagged_to_ukraine_topical_event?` There's only one Ukraine topical event and unlikely to be another. It also makes the content of the method `show_ukraine_cta?` a bit slimmer, so when we have to add some `and also tagged to these taxons` stuff later it will be less clunky.

```suggestion
      def content_tagged_to_ukraine_topical_event?
         ukraine_topical_event_id = "bfa79635-ffda-4b5d-8266-a9cd3a03649c"
```